test/e2e/cmd/workspaces_test.go

```diff
@@ -36,48 +34,106 @@ import (
 const (
 	testResultsDirectory = "/tmp/artifacts"
 	jUnitOutputFilename  = "junit-workspaces-operator.xml"
+	testServiceAccount   = "terminal-test"
 )
 
+func waitDeletionWorkspace() error {
+	var isRemoved = false
+	var err error
+	thresholdAttempts := 10
```

Comment:
on my crc it gets 15 seconds to get namespace fully removed. So, I think we can increase it up to 60 seconds without worry that tests will be slow

Reply:
Done